.gitignore

```diff
@@ -48,6 +48,8 @@ fastlane/report.xml
 fastlane/README.md
 fastlane/screenshots/
 fastlane/test_output/
+# Ignore Fastlane's default/release_notes.txt because the automation copies it from the Simplenote/Resources/release_notes.txt file in
+fastlane/metadata/default/release_notes.txt
```

Comment:
What do you think of this?

My rationale is to have less noisy diffs and honor the (current) single source of truth that is the file in `Simplenote/Resources/`

Reply:
I think we should instead make `fastlane/metadata/default/release_notes.txt` be the new source of truth, and get rid of `Simplenote/Resources/release_notes.txt`.

This will simplify the setup drastically, reducing the number of intermediate files for release notes and the confusion it has always led to.

This is also something we've always wanted to simplify; and is an approach @iangmaia and I have already adopted in Tumblr. So I think that's the right move forward 🙂

The only other things to remember to do when migrating to that solution is to:
 - Update the release scenario accordingly to mention the new location where to update the release notes during code freeze
 - Make the `complete_code_freeze` deletes previous translations from `fastlane/metadata/{any-lang-other-than-default}/release_notes.txt` in parallel of using the `.pot` with the new English copy from `fastlane/metadata/default/release_notes.txt`, so that we don't risk using old release notes from previous version if some locales haven't translated the new ones in time by the time we pull them before submission

Reply:
I think that's the right way forward, too. See also the `FIXME` in the download lane below.

However, I admit that if you hadn't left this comment, implementing it would have been low on my priority list...

I'm a bit unsure about making the switch in the context of this PR, though. I'd feel better about making it happen in one go and test it "end-to-end" on the next code freeze.